### DISCUSSION TITLE
Can we get a new 0.3.2 release?

### DISCUSSION THREAD

**Discussion by ColtonIdle** [whyoleg/cryptography-kotlin#46] [Q&A] [answered]
Looks likes a lot of good new changes have been added. =)

**Comment by whyoleg** [comment, marked as answer]
Hey!
Next release will be 0.4.0.
And yes, there are a lot of good things there. But I need to finalize integration with kotlinx-io and do more testing of it to do a release.
There are several small breaking changes which I've decided to do in one go.
So, stay tuned, hopefully the release will be ready soon-ish.
